src/AzureSqlAction.ts

```diff
@@ -127,9 +127,10 @@ export default class AzureSqlAction {
             case SqlPackageAction.Publish: 
             case SqlPackageAction.Script:
             case SqlPackageAction.DeployReport:
-                args += `/SourceFile:"${inputs.filePath}"`
+                args += `/Action:${SqlPackageAction[inputs.sqlpackageAction]} /TargetConnectionString:"${inputs.connectionConfig.ConnectionString}" /SourceFile:"${inputs.filePath}"`
```

Comment:
Please also add a space before ` /Action` so this doesn't get appended to the last argument value

Reply:
I have a commit up adding that space. My only concern is that before I put up this PR there was no space at the beginning. Because we declare `args` inside of that method and first assign to it inside of that case statement, it makes sense to me why we do not need that space. Do you have any concern with the new leading whitespace in that context?

Thanks!